<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EspecialidadForm</class>
 <widget class="QDialog" name="EspecialidadForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>791</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="descripcion_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripcion :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
